src/main/java/com/commercetools/project/sync/service/impl/CustomObjectServiceImpl.java

```diff
@@ -1,8 +1,7 @@
 package com.commercetools.project.sync.service.impl;
 
-import static com.commercetools.project.sync.util.SyncUtils.getApplicationName;
+import static com.commercetools.project.sync.util.SyncUtils.*;
```

Comment:
Use method name here instead of wildcard :)

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/168/commits/61bdbe0d44a71b11c7eebc923bcea155096d3adf